Tests/UnitTests/Core/TrackFitting/Gx2fTests.cpp

```diff
@@ -291,16 +291,14 @@ std::shared_ptr<const TrackingGeometry> makeToyDetectorYdirection(
   //    volumeConfig2.length = volumeConfig.length;
   volumeConfig2.length = {2 * halfSizeSurface, (nSurfaces + 1) * 1_m,
                           2 * halfSizeSurface};
-  ;
   volumeConfig2.position = {volumeConfig2.length.x(),
                             volumeConfig2.length.y() / 2, 0.};
   volumeConfig2.name = "AdditionalVolume";
   volumeConfig2.binningDimension = Acts::BinningValue::binY;
 
   // Outer volume - Build TrackingGeometry configuration and fill
   CuboidVolumeBuilder::Config config;
-  config.length = {4 * halfSizeSurface, (nSurfaces + 1) * 1_m,
-                   2 * halfSizeSurface};
+  config.length = {10_m, 10_m, 10_m};
```

Comment:
I think we should make the size variable. If we would call `makeToyDetectorYdirection(...)` with `nSurfaces > 10`, not all of them would be covered.

Reply:
I reverted this